force-app/main/default/lwc/donationHistoryFilter/donationHistoryFilter.js

```diff
@@ -6,25 +6,25 @@ import getYearsWithDonation from '@salesforce/apex/DonationHistoryController.get
 const LIFETIME_VALUE = 'lifetime';
 export default class DonationHistoryFilter extends LightningElement {
     
+    @track options = [];
+
+    @api contactId;
```

Comment:
Variable ordering here, generally we like `@api` enabled variables as the top-most.
```suggestion
    @api contactId;

    @track options = [];
```